components/scream/src/dynamics/homme/tests/homme_pd_remap_tests.cpp

```diff
@@ -117,29 +113,31 @@ TEST_CASE("remap", "") {
   std::vector<FieldTag> v_3d_dyn_tags   = {EL,     CMP, GP, GP, VL};
   std::vector<FieldTag> ss_3d_dyn_tags  = {EL, TL,      GP, GP, VL};
   std::vector<FieldTag> vs_3d_dyn_tags  = {EL, TL, CMP, GP, GP, VL};
+  std::vector<FieldTag> tr_3d_dyn_tags  = {EL,     VAR, GP, GP, VL};
```

Comment:
Will all of these references to "VL" need to be replaced now that #897 will be going in?

Reply:
Good point. I'll rebase on master to bring those in.